inference-engine/tests/ie_test_utils/common_test_utils/behavior_test_utils.hpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
It's not a common util! Please move them to the functionalTestUtils

Reply:
Yes